public/index.js

```diff
@@ -33,6 +34,8 @@ buttons.forEach((button) => {
   button.addEventListener("click", () => {
     console.log(lactoseIntolerant.checked);
     dishOriginCountry = button.value;
+    loadingContainer.style.display = "block";
+    resultElement.innerHTML = '';
```

Comment:
This resets the previously viewed recipe to disappear each time to only show the loading indicator. @josueJURE